### DISCUSSION TITLE
How to see the wildcards chosen in the image metadata?

### DISCUSSION THREAD

**Discussion by halkony** [adieyal/sd-dynamic-prompts#788] [Q&A] [answered]
I'm using some wildcards, but the only metadata on my image is the location of the wildcard prompt -- even when I enable the "Save template to metadata" option in the dynamic prompts settings. How can I see what wildcards were chosen/realized for a given image?

I see the prompts are saved to a .txt file, but that seems inconvenient to cross-reference with my results.

**Comment by halkony** [comment]
After some more testing, It's possible my wildcards may not be working correctly. I'm using a wildcard pack in a .yaml I got from the internet, and I'm using the identifier in the wildcards manager in my prompts. I'll have to look into it.

**Comment by halkony** [comment, marked as answer]
The plugin was not working as intended due to a plugin conflict. Disabling other plugins and removing the prompt-based ones I didn't need fixed my issues.

**Reply by ArnorWing** [reply]
Remember which plugin it was causing a conflict?
